README.md

```diff
@@ -825,7 +825,7 @@ Changes not staged for commit:
   (use "git add <file>..." to update what will be committed)
   (use "git checkout -- <file>..." to discard changes in working directory)
 
-	modified:   README.md
+  modified:   README.md
```

Comment:
Shouldn't this read `both modified` if it has conflicts?

Reply:
I originally only corrected whitespace on this line, but I think you're correct and updated to include `both`.